### ISSUE TITLE
Improvement

### ISSUE THREAD

**Issue by jringeisen** [itzikbenh/LaraVue-Datatables#1]
Hey, I just started playing around with this to integrate into my project and was able to get it up and running pretty quickly. I do have a suggestion though. When you enter text into the search box it runs a get request each time a letter is entered, so for instance, if I enter 'programming' it will run 11 requests.

If you add the following it will only run a request when the user is done typing, cutting down on the amount of requests a lot.

```
<input class="form-control" type="text" v-model="tableData.search" placeholder="Search Table"
      @input="search">

methods: {
    search: _.debounce(function (e) {
            this.getProjects()
        }, 700),
}
```

**Comment by itzikbenh** [comment]
Yes, you're correct. If your app is serving many users that would be the way to go. Since this is just a basic tutorial I didn't want to get into these details, but it's still important for others to know about this, so I'll just keep this issue open so others would see. 
Thanks for mentioning this.

**Comment by jringeisen** [comment]
Sounds good, I went ahead and forked the project and made some changes like incorporating Bootstrap. I use datatables a lot so this was a great starting point.

**Comment by Wiich0** [comment]
Have you tried Vuetify!?

**Comment by itzikbenh** [comment]
@Wiich0 No, I haven't.
